src/app/_components/StreamingArea/MiniVideo.client.tsx

```diff
@@ -1,21 +1,52 @@
 'use client';
 
-import useLive from '@/app/_hooks/live/useLive';
-import { useUID } from '@/app/_store/context/useUid';
-import { useState } from 'react';
+import { useEffect, useRef, useState } from 'react';
+import type * as AgoraRTCType from 'agora-rtc-sdk-ng';
 
 interface Props {
   is_active: boolean;
-}
+  videoTrackRef:React.RefObject<AgoraRTCType.ILocalVideoTrack | null>;
+};
 
-const MiniVideo = ({ is_active }: Props) => {
-  const [isHidden, setIsHidden] = useState(false);
+// 미디어 트랙이 있으면 이를 할당한 videoElRef 반환
+const useVideoRef = ({ is_active, videoTrackRef }: Props) => {
+  const videoElRef = useRef<HTMLVideoElement>(null);
+  const limit = useRef<boolean>(false);
+  const SEC = 2 * 1000;
+
+  // 2초 단위로 미디어 스트림 확인
+  useEffect(() => {
+    if (!is_active) {
+      if(videoElRef.current) 
+        videoElRef.current.srcObject = null; // 비디오 src 초기화
+      return;
+    }
```

Comment:
```suggestion
    if (!is_active && videoElRef.current) {
        return videoElRef.current.srcObject = null; // 비디오 src 초기화
    }
```

사소한거긴 하지만 이런식으로 작성하면 좀 더 깔끔할 것 같습니다!

Reply:
요 코드 같은 경우 저렇게 합치면 
useEffect 내부에서 클린업 함수로 판단되서 리턴 값을 무조건 null을 내보내야 한다네요
``` typescript
if (!is_active && videoElRef.current) {
      videoElRef.current.srcObject = null; // 비디오 src 초기화
      return;
}
```
조건만 합쳐서 커밋하겠읍니다!